spec/features/projects/show_spec.rb

```diff
@@ -0,0 +1,97 @@
+    @erin = Contestant.create(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15)
+
+
+    ContestantProject.create(contestant_id: @jay.id, project_id: @news_chic.id)
+    ContestantProject.create(contestant_id: @gretchen.id, project_id: @news_chic.id)
+    ContestantProject.create(contestant_id: @gretchen.id, project_id: @upholstery_tux.id)
+    ContestantProject.create(contestant_id: @kentaro.id, project_id: @upholstery_tux.id)
+    ContestantProject.create(contestant_id: @kentaro.id, project_id: @boardfit.id)
+    ContestantProject.create(contestant_id: @erin.id, project_id: @boardfit.id)
+  end
+
+  it "shows project attributes" do
+    # As a visitor,
+    # When I visit a project's show page ("/projects/:id"),
+    # I see that project's name and material
+    # And I also see the theme of the challenge that this project belongs to.
+    # (e.g.    Litfit
+    #     Material: Lamp Shade
+    #   Challenge Theme: Apartment Furnishings)
+    visit "/projects/#{@news_chic.id}"
+
+    expect(page).to have_content @news_chic.name
+    expect(page).to have_content @news_chic.material
+    expect(page).to have_content @recycled_material_challenge.theme
+  end
+
+  it "shows the number of contestants on project" do
+    visit "/projects/#{@news_chic.id}"
+
+    expect(page).to have_content "Number of Contestants: #{@news_chic.contestants.count}"
+  end
+
+  it "shows the avg years of exp. for contestants by project" do
+    # As a visitor,
+    # When I visit a project's show page
+    # I see the average years of experience for the contestants that worked on that project
+    # (e.g.    Litfit
+    #     Material: Lamp Shade
+    #   Challenge Theme: Apartment Furnishings
+    #   Number of Contestants: 3
+    #   Average Contestant Experience: 10.25 years)
+    visit "/projects/#{@news_chic.id}"
+
+    expect(page).to have_content "Average Contestant Experience: 12.5 years"
+  end
+
+  describe "adds contestant to project" do
+    # As a visitor,
+    # When I visit a project's show page
+    # I see a form to add a contestant to this project
+    # When I fill out a field with an existing contestants id
+    # And hit "Add Contestant To Project"
+    # I'm taken back to the project's show page
+    # And I see that the number of contestants has increased by 1
+    # And when I visit the contestants index page
+    # I see that project listed under that contestant's name
+    it "shows form to add contestant" do
+      visit "/projects/#{@news_chic.id}"
+      
+      expect(page).to have_field :name
+    end
+
+    it "redirects to projects show page, data increased, and reflected on contestants index" do
+      visit "/projects/#{@news_chic.id}"
+
+      fill_in "Name", with: "Sir TP"
+      fill_in "Years of Exp:", with: 3
+      fill_in "Age", with: 33
+      fill_in "Hometown", with: "PHX"
+      click_button "Add Contestant"
```

Comment:
This form would actually look like:
```ruby
fill_in :contestant_id, with: @erin.id

click_button "Add Contestant To Project"
```
We are only looking for an existing contestant to add to this project.